### DISCUSSION TITLE
Is there a way to get a list of contributors(github id) to each page?

### DISCUSSION THREAD

**Discussion by hochan222** [mdn/mdn-community#728] [Platform] [answered]
### Background of the Suggestion

Current contributors can be found through the links below. 

`https://developer.mozilla.org/<locale>/<file-path>/contributors.txt`

The contents are provided below from `renderContributorsTxt` method in yari. Also, `_wikihistory.json`(for Original Wiki contributors) remains legacy.

```
# Contributors by commit history
https://github.com/mdn/translated-content/commits/main/files/ko/web/javascript/reference/global_objects/array/of/index.md

# Original Wiki contributors
alattalatta
KisukPark
limkukhyun
ligeek
````

It is rare for users to check this by going to the link.

### Suggestion

Added to the bottom of the contributor profile.

<img width="540" alt="image" src="https://github.com/user-attachments/assets/d8c89cb1-141b-4c11-84fc-5531e638c7b5">

- https://github.com/mdn/yari/pull/11755

### Problems

There is no way to get the contributor's github ID for each page now. Although the above PR used the github API, there are limitations. Also, We can't get github id with git cli.

**I'm wondering if there is a solution at this point.**

### Solution

(1) If there is no good way to get the github id, I think it's still good.

(2) Added "contributors" to metadata. (+github actions..?)

```
---
title: ~
slug: ~
contributors: ~
---
```

(3) ..?

**Comment by skyclouds2001** [comment]
there is a https://github.com/mdn/content/blob/main/files/en-us/_wikihistory.json, but the data seems not longer updated

**Reply by caugner** [reply]
The wikihistory is showing contributions from before content was in a git repository, when MDN was a wiki powered by [kuma](https://github.com/mdn/kuma).

**Comment by pransh15** [comment, marked as answer]
Hi @hochan222 👋 

I apologize this slipped through the cracks. Thanks for sharing this suggestion! I appreciate you thinking of the community and how we can add contributors to the page they have helped build. ❤️ 

I like your suggestion, and I will take it to the team to determine how we can make it happen. I want you to know that it might not be prioritized for the rest of the year.
